### ISSUE TITLE
Release v0.0.2

### ISSUE THREAD

**Issue by surajpaib** [project-lighter/lighter#118]
Given we are having more users in this repo. It would be nice to have the latest code in the 
`pip install project-lighter` without adding the `--pre` flag.

**Comment by ibro45** [comment]
Alright let's merge https://github.com/project-lighter/lighter/pull/111 and we can do that

**Comment by surajpaib** [comment]
Waiting on #121 before releasing v0.0.3

**Comment by surajpaib** [comment]
@ibro45 I was mistaken, its `v0.0.2` now. 

Released!! Congrats
